npm-js/create-kalix-entity/template/base-ts/tsconfig.json

```diff
@@ -3,8 +3,9 @@
     "outDir": "./dist",
     "strict": true,
     "strictFunctionTypes": true,
+    "skipLibCheck": true, //TODO consider https://classic.yarnpkg.com/en/docs/selective-version-resolutions/
```

Comment:
I don't quite understand the consequences of this, @pvlugter do you?

Problem was that it fails on some type validation in a mongo library which we for some unclear reason depend on

Reply:
Reverted this change to see if the tests are ok now. Tests look fine on main as well.

Reply:
Tests passing fine without this now.